tslint.dot

```diff
@@ -1,8 +1,3 @@
 {
-    "extends": "dtslint/dt.json",
-    "rules": {
-        "interface-name": [false],
-        "ban-types": [false],
-        "await-promise": [true, "Request"]
-    }
+  "extends": "dtslint/dt.json"
 }
```

Comment:
Just to be clear, is this change what is causing unit tests to fail per:

> Note, that tests are currently failing because of this: #116

Reply:
Yes, exactly. I don't want to merge it before we can clean `tslint.dot` anyway, so it's fine.